### ISSUE TITLE
[BUG] HTTPS required

### ISSUE THREAD

**Issue by hoangduy1912** [camunda/camunda-platform-helm#477]
**Describe the bug:**

Our Camunda platform environment is using separated ingress without HTTPS. It works fine on version 8.0 but when I upgraded to 8.1, there's an error when browsing service URL
![image](https://user-images.githubusercontent.com/11615917/199914692-00b29b43-e28d-4868-9cff-af68573c5e05.png)

Log from keycloak pod
09:00:27,910 WARN  [org.keycloak.events] (default task-2) type=LOGIN_ERROR, realmId=master, clientId=null, userId=null, ipAddress=20.4.9.81, error=ssl_required

Am I missing any configuration in the new version?

Here's the custom value I'm using

```
operate:
  service:
    type: NodePort
identity:
  service:
    type: NodePort
  keycloak:
    service:
      type: NodePort
  env:
    - name: IDENTITY_URL
      value: "http://identity.example.com"
tasklist:
  service:
    type: NodePort
optimize:
  image:
    tag: 3.9.1
  service:
    type: NodePort
zeebe-gateway:
  service:
    type: NodePort
global:
  image:
    tag: 8.1.2
  identity:
    auth:
      publicIssuerUrl: "http://keycloak.example.com/auth/realms/camunda-platform"
      operate:
        redirectUrl: "http://operate.example.com"
      tasklist:
        redirectUrl: "http://tasklist.example.com"
      optimize:
        redirectUrl: "http://optimize.example.com"
```

**Environment:**
- Platform: EKS
- Chart version: 8.1

**Comment by aabouzaid** [comment]
Hi @hoangduy1912
This issue has been fixed in chart [v8.1.1](https://artifacthub.io/packages/helm/camunda/camunda-platform/8.1.1)

**Comment by hoangduy1912** [comment]
Thanks @aabouzaid. It worked.
